rmw_connext_dynamic_cpp/src/functions.cpp

```diff
@@ -51,9 +51,11 @@
 #include "rmw/allocators.h"
 #include "rmw/error_handling.h"
 #include "rmw/get_service_names_and_types.h"
+#include "rmw/get_topic_info.h"
 #include "rmw/get_topic_names_and_types.h"
 #include "rmw/init.h"
 #include "rmw/rmw.h"
+#include "rmw/topic_info_array.h"
 #include "rmw/types.h"
 
```

Comment:
Add `#include "rmw_connext_shared_cpp/topic_info.hpp"` too.

Reply:
[This](https://github.com/ros2/rmw_connext/pull/377/files/fed79ad5fd3a79eede8267c6e3f474ee1c24b508#diff-d8fc519447c043a7165e1ff89d1c2088R79) line includes "rmw_connext_shared_cpp/shared_functions.hpp" which includes "topic_info.hpp" ([here](https://github.com/ros2/rmw_connext/pull/377/files/fed79ad5fd3a79eede8267c6e3f474ee1c24b508#diff-4de018e596e156bf378a126997f6cf98R35)) that should effectively do the same right?

Reply:
I think it's more about being explicit about what you're using.

Reply:
Done.

Reply:
Hmm, I agree that being explicit is better. However, the previously defined functions are included from the "shared_functions" header.

Reply:
Oh, I see now that that header is specifically meant for reducing the number of `#include`s in a `.cpp` file. I think it's not worth reverting the change now though.

Reply:
Sounds good.